docs/user_manual/calculations.md

```diff
@@ -659,21 +659,21 @@ Internally, to achieve an optimal regulated tap position, the control algorithm
-| ----------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
-| {py:class}`SearchMethod.scanline <power_grid_model.enum.SearchMethod.scanline>`           | &#10004;                                                                                                | &#10004;                                                                                                    | &#10004;                                                                                                    | &#10008; |
-| {py:class}`SearchMethod.binary_search <power_grid_model.enum.SearchMethod.binary_search>` | &#10008;                                                                                                | &#10004;                                                                                                    | &#10004;                                                                                                    | &#10004; |
+|               | {py:class}`TapChangingStrategy.any_valid_tap <power_grid_model.enum.TapChangingStrategy.any_valid_tap>` | {py:class}`TapChangingStrategy.min_voltage_tap <power_grid_model.enum.TapChangingStrategy.min_voltage_tap>` | {py:class}`TapChangingStrategy.max_voltage_tap <power_grid_model.enum.TapChangingStrategy.max_voltage_tap>` | {py:class}`TapChangingStrategy.fast_any_tap <power_grid_model.enum.TapChangingStrategy.fast_any_tap>` |
+| ------------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| linear search | &#10004;                                                                                                | &#10004;                                                                                                    | &#10004;                                                                                                    | &#10008; |
```

Comment:
If the changes are correct (only expose `TapChangingStrategy` with a fixed search algorithm per strategy), this table becomes obsolete